temfpy/uncertainty_quantification.py

```diff
@@ -1,38 +1,122 @@
-import numpy as np
-import math
-
-
-def borehole(x):
-    """The Borehole function models water flow through a borehole. Its simplicity and quick
-    evaluation makes it a commonly used function for testing a wide variety of methods in
-    computer experiments.
-    """
-    assert len(x) == 8
-
-    r_w = x[0]
-    r = x[1]
-    T_u = x[2]
-    H_u = x[3]
-    T_l = x[4]
-    H_l = x[5]
-    L = x[6]
-    K_w = x[7]
-
-    a = 2 * math.pi * T_u * (H_u - H_l)
-    b = np.log(r / r_w)
-    c = (2 * L * T_u) / (b * r_w ** 2 * K_w)
-    d = T_u / T_l
-
-    rslt = a / (b * (1 + c + d))
-    return rslt
-
-
-def ishigami(x, a=7, b=0.1):
-    """
-    The Ishigami function of Ishigami & Homma (1990) is used as an example for uncertainty and
-    sensitivity analysis methods, because it exhibits strong nonlinearity and nonmonotonicity.
-    """
-    assert len(x) == 3
-
-    rslt = (1 + b * x[2] ** 4) * np.sin(x[0]) + a * np.sin(x[1]) ** 2
-    return rslt
+"""Test capabilities for uncertainty quantification.
+
+This module contains a host of test models and functions often used in the uncertainty
+quantification literate.
+
+"""
+import numpy as np
+import math
+
+
+def borehole(x):
+    """The Borehole function models water flow through a borehole. Its simplicity and quick
+    evaluation makes it a commonly used function for testing a wide variety of methods in
+    computer experiments.
+    """
+    assert len(x) == 8
+
+    r_w = x[0]
+    r = x[1]
+    T_u = x[2]
+    H_u = x[3]
+    T_l = x[4]
+    H_l = x[5]
+    L = x[6]
+    K_w = x[7]
+
+    a = 2 * math.pi * T_u * (H_u - H_l)
+    b = np.log(r / r_w)
+    c = (2 * L * T_u) / (b * r_w ** 2 * K_w)
+    d = T_u / T_l
+
+    rslt = a / (b * (1 + c + d))
+    return rslt
+
+
+def ishigami(x, a=7, b=0.1):
+    """
+    The Ishigami function of Ishigami & Homma (1990) is used as an example for uncertainty and
+    sensitivity analysis methods, because it exhibits strong nonlinearity and nonmonotonicity.
+    """
+    assert len(x) == 3
+
+    rslt = (1 + b * x[2] ** 4) * np.sin(x[0]) + a * np.sin(x[1]) ** 2
+    return rslt
+
+
+def eoq_harris(x, r=10):
+    r"""Economic order quantity model.
+
+    This function computes the optimal economic order quantity (EOQ) based on the model presented in
+    [1]_. The EOQ minimizes the holding costs as well as ordering costs. The core parameters of
```

Comment:
![Screenshot from 2020-05-22 10-42-00](https://user-images.githubusercontent.com/3607591/82649041-e5657080-9c18-11ea-960f-1038554ec2bd.png)

The numbers show up kind of weird in the compiled version, see attached. Is there any configurations we can set that they look normal, i.e. just a simple `[1]` on the line now they are lookig like a superscript. I know that this was already the case when I handed it over to you, but please see if you can find a better solution.

Reply:
I think this is because one reference cannot be referred to twice…? Let me try to fix it.

Reply:
Yes, yes.

Reply:
Great, thanks!